server/controller/requestController.js

```diff
@@ -0,0 +1,78 @@
+import pg from 'pg';
+import jwt from 'jsonwebtoken';
+
+import developmentConfig from './../config/developmentConfig';
+import testConfig from './../config/testConfig';
+
+let config;
+
+if (process.env.NODE_ENV === 'development') {
+  config = developmentConfig;
+} else {
+  config = testConfig;
+}
+
+const client = new pg.Client(config);
+client.connect();
+require('dotenv').config();
+
+/**
+ * it is a class that control all a requests method
+ */
+class RequestController {
+  /**
+ * it gets all users requests
+ * @param {string} req
+ * @param {string} res
+ * @return {object} an object
+ */
+  static async getRequests(req, res) {
+    try {
+      const token = req.headers['x-access-token'] || req.body.token || req.query.token;
+
+      const decoded = await jwt.verify(token, process.env.SECRET);
+      const { id } = decoded;
+
+      const getAUserQuery = `
+            SELECT * 
```

Comment:
Specify the exact fields you want from the user query instead of specifying all.